src/app_service/service/accounts/dto/wallet_secretes_config.nim

```diff
@@ -0,0 +1,41 @@
+import json
+
+type
+  WalletSecretsConfig* = object
+    poktToken*: string
+    infuraToken*: string
+    infuraSecret*: string
+    openseaApiKey*: string
+    raribleMainnetApiKey*: string
+    raribleTestnetApiKey*: string
+
+    # ganacheURL: string // WARNING: is this used?
```

Comment:
I'm not sure if they use it still cc @alaibe

Reply:
Nope, it shouldn't be it was used for test i believe